[2020-01-04 18:56:43,818] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: my_dag.id_scrape 2019-01-04T20:10:00+00:00 [queued]>
[2020-01-04 18:56:43,868] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: my_dag.id_scrape 2019-01-04T20:10:00+00:00 [queued]>
[2020-01-04 18:56:43,868] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-01-04 18:56:43,868] {taskinstance.py:867} INFO - Starting attempt 1 of 1
[2020-01-04 18:56:43,869] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-01-04 18:56:43,915] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): id_scrape> on 2019-01-04T20:10:00+00:00
[2020-01-04 18:56:43,920] {standard_task_runner.py:52} INFO - Started process 22229 to run task
[2020-01-04 18:56:44,054] {logging_mixin.py:112} INFO - [2020-01-04 18:56:44,053] {dagbag.py:403} INFO - Filling up the DagBag from /home/stevo517/web-scraper/airflow-home/dags/my_dag.py
[2020-01-04 18:56:44,154] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: my_dag.id_scrape 2019-01-04T20:10:00+00:00 [running]> web-scrape-test
[2020-01-04 18:56:44,228] {python_operator.py:105} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=id_scrape
AIRFLOW_CTX_EXECUTION_DATE=2019-01-04T20:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-04T20:10:00+00:00
[2020-01-04 18:56:44,228] {taskinstance.py:1088} ERROR - 'function' object has no attribute 'Scrape'
Traceback (most recent call last):
  File "/home/stevo517/anaconda3/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 955, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/stevo517/anaconda3/lib/python3.7/site-packages/airflow/operators/python_operator.py", line 113, in execute
    return_value = self.execute_callable()
  File "/home/stevo517/anaconda3/lib/python3.7/site-packages/airflow/operators/python_operator.py", line 118, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "../scrape_for_dag.py", line 4, in scrape
    my_scrape = scrape.Scrape()
AttributeError: 'function' object has no attribute 'Scrape'
[2020-01-04 18:56:44,230] {taskinstance.py:1119} INFO - Marking task as FAILED.
[2020-01-04 18:56:53,741] {logging_mixin.py:112} INFO - [2020-01-04 18:56:53,740] {local_task_job.py:103} INFO - Task exited with return code 1
